### PR TITLE
📄 Content - Add Caption Fields

### LINKED ISSUE #1620: 💸 Videos - should show title and duration as a figure
Videos currently do not have a duration visible

As per https://ssw.com.au/rules/add-useful-and-concise-figure-captions/ they should.

**Describe the solution you'd like**

Videos should look like they follow the rule

```[tasklist]
### Tasks
- [x] Add video title and duration to video to video component (mandatory)
- [x] Migrate old data - look up the video and update the content
```

### PR DESCRIPTION
**Pull request by Harry-Ross**
- Affected routes: `/consulting/*`, `/offices/*`

- Fixed #1620

- [ ] If adding a new page, I have followed the [📃 New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) issue template

- [x] Include done video or screenshots

image
**Figure: Video caption on `/consulting/react`**

image
**Figure: Editing the video caption on `/admin`**

### CODE REVIEW

File: components/blocks/videoEmbed.tsx [hunk 1]
```diff
@@ -27,6 +28,12 @@ export const VideoEmbed = ({ data }: VideoEmbedProps) => {
       )}
     >
       <VideoModal url={data.url} overflow={data.overflow} />
+      {data.caption && (
+        <p className={classNames("font-bold", !uncentre && "text-centre")}>
+          {data.caption}
```
Comment (wicksipedia) [inline components/blocks/videoEmbed.tsx]: Given that the rule https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions says that the figure should always start with `Video:`

should we hardcode this and strip it from the incoming caption?

**Review by wicksipedia — APPROVED** [review]
minor comment but lgtm

File: components/blocks/videoEmbed.tsx (hunk 1 above)
Comment (Harry-Ross) [reply components/blocks/videoEmbed.tsx]: Yeah, good idea. Do you think maybe there should be a separate string field for duration as well?

Comment (wicksipedia) [reply components/blocks/videoEmbed.tsx]: yep, that should prompt people to put in the duration then

**Pull request merged**